### PR TITLE
Add Registering page

### PR DESCRIPTION
**Pull request by solring**
NOTE: I also move some common styles into Page.module.css and delete HomePage css module since it's not needed anymore.

### CODE REVIEW

File: src/components/RegisterPage.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  // -------- Component Instance ----------
+  return (
+    <div className={pageStyles['root']}>
+      <div className='container vh-100'>
+        <div className="row">
+          <div className="col-12 col-md-6 mx-auto">
+
+            <div className="form-group">
+              <label htmlFor="accountField">帳號</label>
+              <input id="accountField" className="form-control " type="text" placeholder="Username:" aria-label="Username" value={username} onChange={(e) => changeUsername(e.target.value)}/>
```
Comment (chhsiao1981) [inline src/components/RegisterPage.js:77]: I don't think we setup "id" in react-js～
react-js automatically determines the id.

Comment (chhsiao1981) [reply src/components/RegisterPage.js:77]: https://reactjs.org/docs/accessibility.html
https://stackoverflow.com/questions/12894169/what-is-the-html-for-attribute-in-label

**Comment by chhsiao1981** [comment]
Thanks for the PR～

The UI layout looks great～
I'll merge the PR and tweak the details of the reducers later～

**Comment by solring** [comment]
Thank you @chhsiao1981 ! 
Also I saw your comments about input id. So is it OK to use the traditional way to label the input for accessibility?

**Pull request merged**